libc-test/build.rs

```diff
@@ -3964,6 +3964,12 @@ fn test_linux(target: &str) {
             if loongarch64 && (name == "MFD_NOEXEC_SEAL" || name == "MFD_EXEC") {
                 return true;
             }
+            // FIXME: Requires musl >= 1.2
```

Comment:
Could you make this `FIXME(musl)`? Trying to keep all our fixmes greppable since we have so many of them